textproc/jet/Portfile

```
@@ -0,0 +1,67 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        borkdude jet 0.6.26 v
+github.tarball_from releases
+revision            0
+
+maintainers         {@catap korins.ky:kirill} openmaintainer
+license             EPL-1
+categories          textproc devel
+
+description         CLI to transform between JSON, EDN, YAML and Transit using Clojure
+long_description    {*}${description}
+
+distfiles           ${name}-${version}-standalone.jar
+
+checksums           rmd160  7730a89dbb9117d066a633aad51cfbad6a6d5a43 \
+                    sha256  ce5488219d1857abacb20032973f101a62818bc3302083339a5a087f86c7bc75 \
+                    size    13531702
+
+extract {}
```

Rather than overriding the extract phase, tell MacPorts there's nothing to extract:
```suggestion
extract.only
```

---

thanks!